Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Base image 
+FROM python:3.8-slim-buster
+
+# Copy the current directory contents into the container
+
+COPY . /travel
+
+#Set the working directory in the container to travel
+
+WORKDIR /travel
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    curl \
+    software-properties-common \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the required dependencies 
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE 8501
+
+HEALTHCHECK CMD curl --fail http://localhost:8501/_stcore/health
+
+# Define the command to run the app when the container starts
+
+ENTRYPOINT ["streamlit", "run", "app.py", "--server.port=8501", "--server.address=0.0.0.0"]
+
+# Docker file created in the reference of
+# https://docs.streamlit.io/knowledge-base/tutorials/deploy/docker
```

Comment:
Abe avu badhu should be at the top etle je bi dockerfile khole ene khabar pade ke guilde reference kari che. So in case dockerfile work naa kare, they can read up the guide.

Reply:
And BC commit naam etlu kharab kem che [updated the docker file](https://github.com/prayagnshah/travel-advisory/pull/1/commits/773a9cc5a20336c5f3f97307fd45c4f8b3c445a4)? Su _update_ hati?

Reply:
Corrected the mistake!